### PR TITLE
Add fprintd lib

### PR DESCRIPTION
**Pull request by nhubaotruong**
For this issue: https://github.com/flathub/io.ente.auth/issues/4

### CODE REVIEW

**Comment by nhubaotruong** [comment]
bot, build

**Comment by nhubaotruong** [comment]
Hi @prateekmedia, fingerprint unlock is working now, would you mind take a look please ? 
image

File: io.ente.auth.yml [hunk 1]
```diff
@@ -1,24 +1,28 @@
...
-runtime-version: '47'
+runtime-version: "47"
 sdk: org.gnome.Sdk
 command: enteauth
 finish-args:
-  # X11 + XShm access
   - --share=ipc
   - --socket=wayland
   - --socket=fallback-x11
-  - --socket=pulseaudio
```
[the remark this answers is not in the document]
Comment (nhubaotruong) [reply io.ente.auth.yml:11]: This app doesn't looks like it needs access to audio, correct me if I'm wrong

[the remark this answers is not in the document]
Comment (prateekmedia) [reply io.ente.auth.yml:11]: Seems right then.

**Review by prateekmedia — APPROVED** [review]
LGTM!

**Pull request merged**